src/RefractDataStructure.cc

```diff
@@ -41,6 +41,9 @@ namespace drafter {
         if (ta & mson::FixedTypeAttribute) {
             attr->push_back(refract::IElement::Create(SerializeKey::Fixed));
         }
+        if (ta & mson::NullableTypeAttribute) {
+            attr->push_back(refract::IElement::Create(SerializeKey::Nullable));
+        }
 
```

Comment:
Please remove all the changes below this line in this file. They are here because of a bad rebase.